daemon/daemon.go

```diff
@@ -442,7 +442,17 @@ configRetry:
 		log.Infof("Starting the Typha connection")
 		err := typhaConnection.Start(context.Background())
 		if err != nil {
-			log.WithError(err).Fatal("Failed to connect to Typha")
+			retry := 0
```

Comment:
Please log once here at Error level "Failed to connect to Typha, will retry..."

Reply:
Ack